main/lsp/LSPTypecheckerCoordinator.cc

```diff
@@ -30,22 +32,18 @@ void LSPTypecheckerCoordinator::asyncRunInternal(shared_ptr<core::lsp::Task> tas
     }
 }
 
-void LSPTypecheckerCoordinator::asyncRun(function<void(LSPTypechecker &)> &&lambda) {
```

Comment:
Note: This method was unused. Only slow path typechecking and initialization require the ability to run part of their operation asynchronously, and I think it's better to special-case them and carefully handle threading to avoid footguns.